test/katello_test_helper.rb

```diff
@@ -1,17 +1,24 @@
 require 'simplecov'
 require 'simplecov-rcov'
-SimpleCov.formatters = [
-  SimpleCov::Formatter::RcovFormatter,
-  SimpleCov::Formatter::HTMLFormatter
-]
-SimpleCov.start 'rails'
 
 require 'test_helper'
 require 'factory_girl_rails'
 require "webmock/minitest"
 require "mocha/setup"
 require 'set'
 
+SimpleCov.formatters = [
+  SimpleCov::Formatter::RcovFormatter,
+  SimpleCov::Formatter::HTMLFormatter
+]
+
+SimpleCov.start do
+  filters.clear
+  add_filter do |src|
+    !(src.filename =~ /^#{SimpleCov.root}/) unless src.filename =~ /katello/
```

Comment:
@daviddavis This doesn't work for some reason. [Here's the result](http://ci.theforeman.org/job/test_katello_pr_core/2930/database=postgresql,ruby=2.2/rcov/). It's still getting foreman files as well as gems. I'm going to try a simple `=~ /\/katello\//` since all of our katello classes are namespaced under `Katello::`